lib/src/Uncertainty/Bayesian/RandomWalkMetropolisHastings.cxx

```diff
@@ -284,7 +284,13 @@ void RandomWalkMetropolisHastings::load(Advocate & adv)
   adv.loadAttribute("calibrationStrategy_", calibrationStrategy_);
   adv.loadAttribute("samplesNumber_", samplesNumber_);
   adv.loadAttribute("acceptedNumber_", acceptedNumber_);
-  adv.loadAttribute("currentLogLikelihood_", currentLogLikelihood_);
+  if (adv.hasAttribute("currentLogLikelihood_"))
+  {
+    adv.loadAttribute("currentLogLikelihood_", currentLogLikelihoodPlusLogPrior_);
+    LOGWARN(OSS() << "The internals of the class RandomWalkMetropolisHastings have changed in version 1.16. Please save this object again to ensure compatibility with future versions.");
```

Comment:
please drop this warning, it should be silent

Reply:
Done.